source/SpinalHDL/Data types/bits.rst

```diff
@@ -92,29 +97,41 @@ Logic
    * - x.andR
      - AND all bits of x
      - Bool
-   * - x \>\> y
-     - Logical shift right, y: Int
+   * - | y = 1 // int
```

Comment:
This is up there now... https://dlmiles.github.io/SpinalDoc-RTD/docs_multi/html/vTEST/SpinalHDL/Data%20types/bits.html#logic  apart from the extra whitespace towards bottom of cell I think it look better.

I have fixed the use of `int` instead of `Int` to draw attention to types involved.

Reply:
While the newlines are now fixed, to some other things remain:
- The table entries now take up 3 lines (2 lines of content, one empty line)
- "int" is not a type, the correct scala type is "Int"
- I still think it makes it less readable to restate all information twice, both the type of "y" and the size of the output result are present